packages/workflow/src/workflow.ts

```diff
@@ -1253,3 +1253,20 @@ export function upsertSearchAttributes(searchAttributes: SearchAttributes): void
 
 export const stackTraceQuery = defineQuery<string>('__stack_trace');
 export const enhancedStackTraceQuery = defineQuery<EnhancedStackTrace>('__enhanced_stack_trace');
+
+const loggerSinks = proxySinks<LoggerSinks>();
+
+/**
+ * Default workflow logger.
+ * This logger is replay-aware and will omit log messages on workflow replay.
+ * The messages emitted by this logger are funnelled to the worker's `defaultSinks`, which are installed by default.
```

Comment:
```suggestion
 * The messages emitted by this logger are funnelled to the worker's {@link defaultSinks}, which are installed by default.
```

Reply:
I'll try that and see if it gets linked properly, defaultSinks is in a different package.